### PR TITLE
Olli meeco issue credentials

### PR DESCRIPTION
**Pull request by OlliMeeco**
review na ons gesprek van vandaag: heb generate / sign gesplitst

### CODE REVIEW

File: issue-credentials.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+# Issue Credentials
+Issue credential is the first operation in the lifecycle of a [Verifiable Credential](https://www.w3.org/TR/vc-data-model/#lifecycle-details). It assists the Issuer in creating the credential so it can be delivered to the subject of the credential.
+The supported modes are:
```
Comment (javereec) [inline issue-credentials.md:4]: New line before

File: issue-credentials.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+* Generating credential – generates an (unsigned) credential that requires signing.
+* Issuing credential – credential is generated and signed with keys managed on the platform.
+
+The response is a JSON formatted verifiable credential with a system-generated unique id and issuance date.
+
+## Prerequisite
```
Comment (javereec) [inline issue-credentials.md]:
```suggestion
## Prerequisites
```

File: issue-credentials.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+* Claims about the Subject
+* Expiration date
+
+**Responses**
+
+An unsigned credential in JSON format with 
+* id (of the credential)
+* credentialStatus
+* type 
+* issuanceDate
```
Comment (javereec) [inline issue-credentials.md]: Can you make this consistent with wording in https://docs.meeco.me/guides/credential-schemas#create-credential-schema

Below is not what we return (as we return it as vc-jwt). Check with api and Linas

File: issue-credentials.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+## Who can use this?
+An onboarded organisation in the tenacy set up as issuer by the tenant.
+
+## Additional considerations
+VC ids are stored in a Verifiable Data Registry which acts as an inventory of identifiers and schemas. The  registry 'StatusList2021' allows to write and read the status of the credential (ie. revoked, suspended). 
+
+Other examples of verifiable data registries include trusted databases, decentralized databases, government ID databases, and distributed ledgers.
+
+## Generate a Credential
+Generating a credential requires metadata about the credential (type, issue date, issuer) and its subject (subject DID, claims) to be structured following the datamodel [W3C VC-Core](https://www.w3.org/TR/vc-data-model/#core-data-model) and respect the structure of the chosen credential schema. The endpoint will fit the data into the schema and performs a number of coherency checks. The result is a JSON ready to be signed.
```
Comment (javereec) [inline issue-credentials.md]: - Issuer is not meta data and would be more specific (e.g. issuer DID). Use "issuance data" like in the standard
- The data will not really be fitted into the schema. It needs to be conformant with the schema (which we will check).
- Write active (no "will")

File: issue-credentials.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+      "name": [{
+        "value": "SVX"
+      }]
+    }
+  }
+}
+```
+
+## Sign a Credential
+By signing a credential an entity adds proof to a generated credential, and thus the entity becomes the 'issuer' of the crendential. This allows another entity (the 'verifier') to trust the provenance of the crendential. The proof is established by signing a hash of the VC with the issuer's private key.
```
Comment (javereec) [inline issue-credentials.md]:
```suggestion
## Issue Credential

[Generate](#generate-credential) and sign a credential for the given issuer. The result is a signed vc-jwt.

To sign the credential, a fully qualified DID string or object with `id` property needs to be provided. The DID and its verification keypair needs to be under control of the platform to be able to perform the signing.
```

File: issue-credentials.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+POST	/credentials/issue
+```
+
+**Request**
+
+* type – Encryption algorythm
+* private key - from the issuers's contolled assymetic keypair
+
+**Responses**
+A signed credential JSON format including:
```
Comment (javereec) [inline issue-credentials.md]: Same comment as above

File: issue-credentials.md (hunk 3 above)
Comment (OlliMeeco) [reply issue-credentials.md]: "consistent with wording" this is vague to me . I reviewed "credential schema" and can only conclude that the wording is shorter, so i shortened. The ins/outs are different for a schema so i can't reuse those.

**Review by javereec — APPROVED** [review]

**Pull request merged**